packages/portfolio-contract/src/network/network-spec.ts

```diff
@@ -16,7 +16,12 @@ export type TransferProtocol =
   | 'cctpReturn'
   | 'cctpSlow'
   | 'local';
-export type FeeMode = 'toUSDN' | 'gmpCall' | 'gmpTransfer';
+export type FeeMode =
+  | 'toUSDN'
+  | 'makeEvmAccount'
+  | 'cctpReturn'
+  | 'supply'
+  | 'withdraw';
```

Comment:
Please document these values, and avoid reusing "cctpReturn" as both a TransferProtocol and FeeMode. Also, I'm not seeing a difference between "supply" vs. "withdraw", which makes me think they're actually a single mode.
```suggestion
/**
 * Steps submitted to the contract are expected to include fee/gas payment
 * details which vary by the traversed link.
 * - toUSDN: transferring into USDN transfer reduces the *payload* (e.g., $10k
 *   might get reduced to $9995)
 * - cosmosToEvmViaAxelar: a fee for the Axelar GMP transaction (cf.
 *   https://www.axelar.network/blog/general-message-passing-and-how-can-it-change-web3
 *   ) plus gas for our EVM contract
 * - evmToCosmosViaAxelar: a fee for the Axelar GMP transaction based on the EVM
 *   chain
 * - withinEvmViaAxelar: a fee for the Axelar GMP transaction based on the
 *   source and destination chains
 */
export type FeeMode =
  | 'toUSDN'
  | 'cosmosToEvmViaAxelar'
  | 'evmToCosmosViaAxelar'
  | 'withinEvmViaAxelar';
```